lib/logstash/outputs/elasticsearch.rb

```diff
@@ -425,6 +414,18 @@ def generate_jks cert_path
   def receive(event)
     return unless output?(event)
 
+    if @manage_template
```

Comment:
Since there's been no activity here I'm going to close this issue. Thanks for he patch @Bradox , if you'd like to try to revive this idea I'd recommend opening an issue as a design discussion to suss out the pitfalls here!